config/environments/test.rb

```diff
@@ -20,6 +20,9 @@
     'Cache-Control' => 'public, max-age=3600',
   }
 
+  # Ensure that our tests are accounting for caching behaviour
+  config.cache_store = :memory_store
```

Comment:
Do we have facilities for testing this? eg hit a fake API endpoint, render the page, change what's in the API, check the page is unchanged, clear the cache, check the page is changed?

Reply:
Will put something together 👍

Reply:
@duncanjbrown added a spec, wdyt?